app/scripts/controllers/citationInfo.ctrl.js

```diff
@@ -1,192 +1,196 @@
 'use strict';
 
-angular.module('yourStlCourts').controller('CitationInfoCtrl', function (faqData,$state, $window, citations,municipalities,courts,Courts,Session,moment,$anchorScroll,LegalRights) {
+angular.module('yourStlCourts').controller('CitationInfoCtrl', function (faqData, $state, $window, citations, municipalities, courts, Courts, Session,
+                                                                         moment, $anchorScroll, LegalRights) {
   var ctrl = this;
-  var PLACEHOLDER_DL_NUM = "NO_DL_NUM";
+  var PLACEHOLDER_DL_NUM = 'NO_DL_NUM';
 
   ctrl.faqData = faqData;
   ctrl.selectedCitation = null;
-  ctrl.paymentUrl = "";
+  ctrl.paymentUrl = '';
   ctrl.citationCourtLocations = {};
   ctrl.groupedCitations = {};
 
-  ctrl.groupCitationsByDL = function(){
-    var dlNum = "";
-    var dlState = "";
+  ctrl.groupCitationsByDL = function () {
+    var dlNum = '';
+    var dlState = '';
     var groupedCitations = {};
 
-    citations.forEach(function(citation){
+    citations.forEach(function (citation) {
       dlNum = citation.drivers_license_number;
       dlState = citation.drivers_license_state;
-      if (dlNum === ""){
+      if (!dlNum) {
         //in the event the defendant does not have a DL Num assign one so the object has a key
         dlNum = PLACEHOLDER_DL_NUM;
       }
-      if (!groupedCitations[dlNum+dlState]){
-        groupedCitations[dlNum+dlState] = [];
+      if (!groupedCitations[dlNum + dlState]) {
+        groupedCitations[dlNum + dlState] = [];
       }
-      groupedCitations[dlNum+dlState].push(citation);
+      groupedCitations[dlNum + dlState].push(citation);
     });
 
     return groupedCitations;
   };
 
-  ctrl.issueMultiplePeopleWarning = function(){
-
+  ctrl.issueMultiplePeopleWarning = function () {
     return (_.size(ctrl.groupedCitations) > 1);
   };
 
-  ctrl.selectCitation = function(citation,idToScrollTo){
+  ctrl.selectCitation = function (citation, idToScrollTo) {
     ctrl.selectedCitation = citation;
     Session.setSelectedCitation(ctrl.selectedCitation);
-    if (ctrl.selectedCitation) {
```

Comment:
Why did you remove ctrl.selectedCitation check?

Reply:
If you'll see the new initialize logic, I no longer allow the selected citation to be null/undefined so the check was no longer necessary